Description: Include opt/mbsebbs in Makefile.global.in paths.
 Add the path opt/mbse/ before the directories for BINDIR, ETCDIR,
 VARDIR, and INTDIR.  Set DOCDIR to usr/share/doc/mbsebbs.
 .
 This is for the Debian package build.
Author: Robert James Clay <jame@rocasa.us>

Index: mbsebbs-0.92.0/Makefile.global.in
===================================================================
--- mbsebbs-0.92.0.orig/Makefile.global.in	2007-05-27 10:12:52.000000000 -0400
+++ mbsebbs-0.92.0/Makefile.global.in	2012-03-04 16:42:59.000000000 -0500
@@ -10,11 +10,11 @@
 # for make install, where to put binaries and what owner to set
 SUBDIRS		= @SUBDIRS@
 PREFIX		= @prefix@
-BINDIR		= @prefix@/bin
-ETCDIR		= @prefix@/etc
-DOCDIR		= @prefix@/doc
-VARDIR		= @prefix@/var
-INTDIR		= @prefix@/share/int
+BINDIR		= @prefix@/opt/mbse/bin
+ETCDIR		= @prefix@/opt/mbse/etc
+DOCDIR		= @prefix@/usr/share/doc/mbsebbs
+VARDIR		= @prefix@/opt/mbse/var
+INTDIR		= @prefix@/opt/mbse/share/int
 OWNER		= @OWNER@
 GROUP		= @GROUP@
 ROWNER		= @ROWNER@
